manifests/application.pp

```diff
@@ -24,11 +24,14 @@
   include certs::candlepin
   include certs::foreman
   include certs::pulp_client
-  include certs::qpid
   include katello::params
 
-  include katello::qpid_client
-  User<|title == $foreman::user|>{groups +> 'qpidd'}
+
+  if $facts['os']['release']['major'] == '7' {
+    include certs::qpid
+    include katello::qpid_client
+    User<|title == $foreman::user|>{groups +> 'qpidd'}
+  }
 
   foreman_config_entry { 'pulp_client_cert':
```

Comment:
Is this Pulp 2 only or both Pulp 2 & 3? If 2-only, this IMHO also needs conditionals, as well as the inclusion of `certs::pulp_client`.

Reply:
This is both.